lib/signs/utilities/audio.ex

```diff
@@ -365,6 +365,10 @@ defmodule Signs.Utilities.Audio do
       sign,
       Enum.map(audios, &Content.Audio.to_params(&1)),
       Enum.map(audios, &Content.Audio.to_tts(&1)),
+      case audios do
+        [%PaMessages.PaMessage{priority: priority}] -> priority
+        _ -> 2
```

Comment:
In what situations do we have multiple `audios`? Is it okay that we're then ignoring their individual priorities?

Reply:
For PA messages, there's only ever one at a time, so this should be sufficient. A handful of other code paths can generate multiple.